crates/zksync/core/src/vm/inspect.rs

```diff
@@ -394,19 +391,6 @@ fn inspect_inner<S: ReadStorage>(
 
     formatter::print_vm_details(&tx_result);
 
-    info!("=== Console Logs: ");
-    let log_parser = ConsoleLogParser::new();
-    let console_logs = log_parser.get_logs(&call_traces, true);
```

Comment:
I think we want to keep this bit - to display console logs whtn we display the era_test_node execution report, it's useful sometimes